src/controllers/DrawingModeController.js

```diff
@@ -104,7 +113,18 @@ class DrawingModeController extends AnnotationModeController {
         /* eslint-enable require-jsdoc */
 
         this.drawingStartHandler = this.drawingStartHandler.bind(this);
-        this.pushElementHandler(this.annotatedElement, ['mousedown', 'touchstart'], this.drawingStartHandler);
+        this.pushElementHandler(
+            this.annotatedElement,
+            'click',
+            (event) => {
+                const el = findClosestElWithClass(event.target, CLASS_ANNOTATION_POINT_MARKER);
+                if (el) {
+                    event.stopPropagation();
+                }
+            },
+            true
+        );
+        this.pushElementHandler(this.annotatedElement, ['mousedown', 'touchstart'], this.drawingStartHandler, true);
```

Comment:
both are needed for the surface edge cases and click is separate because it more or less just prevents any click events other than drawing ones from ever triggering.